students/jemielity_kamil/lesson_08_regular_expressions/postal_code_validator.py

```diff
@@ -0,0 +1,18 @@
+import re
+
+
+def postal_code_validator(code):
+    regex = re.compile(r'\d{2}-\d{3}')
+    mo = regex.search(code)
```

Comment:
regex.search() will search for substring  but here we want to match the string
Otherwise accepted postal code is: 'bla11bla71-111'